### ISSUE TITLE
Use serif font for passwords and possibly usernames

### ISSUE THREAD

**Issue by bjoern-tantau** [hegocre/NextcloudPasswords#100]
### Checklist

- [X] I have checked and there are *no existing issues* - [open](https://github.com/hegocre/NextcloudPasswords/issues) or [closed](https://github.com/hegocre/NextcloudPasswords/issues?q=is%3Aissue+is%3Aclosed) - which request the same feature.
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.

### Feature description

It would be nice to have a serif font where you can more easily discern an uppercase I from a lowercase l.

### Why do you want this feature?

This is needed when I have to type in a password manually into another device.

### Additional information

Maybe this should be done for all fields. Although people should know their usernames that isn't always the case.

**Comment by hegocre** [comment]
Hello @bjoern-tantau, thank you for your feedback.

For passwords, the app already uses DejaVu Sans Mono, whic [according to Wikipedia](https://en.m.wikipedia.org/wiki/DejaVu_fonts#DejaVu_Sans_Mono), is ideal for this case.

Below I attach a screenshot from the app, with a password that clearly tells appart 1 (one) from l (lowercase L) and I (uppercase i), as well as 0 (zero) from O (letter):
![Screenshot_2024-04-10-20-49-26-16_8e0c5536e9a65f72540ddfa131e9f5fd.jpg](https://github.com/hegocre/NextcloudPasswords/assets/15657088/4b2b8efd-08db-4311-87c2-f2354c2dde14)

I don't think it would be necessary to also do it on usernames, since they don't usually are random like passwords, but feel free to debate me on this.

Cheers!

**Comment by bjoern-tantau** [comment]
I just noticed for the first time that the password is revealed by clicking on it. I always revealed it by clicking edit and then on the eye-icon. So I guess I would want to have the font also be used in the edit-view.

**Comment by hegocre** [comment]
I don't think that would make too much sense, since when typing a password you know what you are writing, and to check a password its easier to just go to the details and reveal it. It is the way that the Nextcloud app behaves, so I think it makes sense to leave it like that. Maybe a way to showcase the tap to reveal feature would be nice, since its not clear now, but I'm not sure what would be the best way.

Best regards!

**Comment by bjoern-tantau** [comment]
> Maybe a way to showcase the tap to reveal feature would be nice, since its not clear now, but I'm not sure what would be the best way.

Maybe put an eye-icon next to the copy-icon.
